routes/club.js

```diff
@@ -1,9 +1,10 @@
 const express = require("express");
 const router = express.Router();
-
-const { editProfile, profile } = require("../helpers/club")
+const { editProfile, profile ,image} = require("../helpers/club")
 
 router.post("/profile", editProfile)
 router.get("/profile", profile)
 
+router.post("/profile/image",image)
```

Comment:
Still a little confused as to why we don't just pass the `authorize` middleware in here like so: 
`router.post("/profile", authorize, editProfile)` 
instead of doing it inside the function definitions themselves, as we currently are. But it's not a big deal. Just keep it in mind for the future!